### DISCUSSION TITLE
Mini-hackathons

### DISCUSSION THREAD

**Discussion by weex** [OpenHospitalityNetwork/fedi-trustroots#68] [General]
In the last meeting I brought up the idea of having a mini-hackathon after a future meeting and everyone seemed receptive so I'm creating this thread to propose that we do one soon. Any suggestions on when? This thread can be used to coordinate the what, where, and when.

**Comment by chagai95** [comment]
I'm in whenever.

**Reply by weex** [reply]
Great, I think we should give this idea a week or two to breathe so the earliest would be Oct 25th. We can try working on stuff after the next couple meetings to see if it should really be a separate time.

**Comment by mariha** [comment]
Depending on the format, I think the best would be during the weekend, either Sat or Sun or both.

30-31 Oct does not work for me, otherwise I am open to whenever works for others.

Initially I thought one of the evenings 9pm-12pm might be ok, but I guess more time and fresher mind would be better?

**Reply by weex** [reply]
Sunday's are a bit easier for me, but I also don't mind shorter events if they benefit from being coincident with or after a meeting. I've enjoyed the growing activity around the meeting time and it felt natural to support with a follow-on event. That time can function as technical office hours and for help onboarding new contributors.
